app/controllers/research_projects_controller.rb

```diff
@@ -0,0 +1,30 @@
+class ResearchProjectsController < ApplicationController
+
+
+  DEFAULT_FUNDER_TYPE = "H2020"
+
+  def index
+    render json: research_projects
+  end
+
+  def search
+    @results = research_projects.select { |r| r.description.match(params[:description]) }
+    logger.debug("Returning #{@results.count} results")
```

Comment:
This log statement doesn't have much context. Either delete it (if it was just there for dev debugging purposes) or add some more context